benches/reader.rs

```diff
@@ -38,3 +43,95 @@ fn record_reader_10k_collect(bench: &mut Bencher) {
     let _ = iter.collect::<Vec<_>>();
   })
 }
+
+#[bench]
+fn record_reader_stock_collect(bench: &mut Bencher) {
+  let path = Path::new("data/stock_simulated.parquet");
+  let file = File::open(&path).unwrap();
+  let len = file.metadata().unwrap().len();
+  let parquet_reader = SerializedFileReader::new(file).unwrap();
+
+  bench.bytes = len;
+  bench.iter(|| {
+    let iter = parquet_reader.get_row_iter(None).unwrap();
+    let _ = iter.collect::<Vec<_>>();
+  })
+}
+
+#[bench]
+fn record_reader_stock_column(bench: &mut Bencher) {
+  /// WARNING THIS BENCH IS INTENDED FOR THIS DATA FILE ONLY
+  /// COPY OR CHANGE THE DATA FILE MAY NOT WORK AS YOU WISH
+  let path = Path::new("data/stock_simulated.parquet");
+  let file = File::open(&path).unwrap();
+  let len = file.metadata().unwrap().len();
+  let parquet_reader = SerializedFileReader::new(file).unwrap();
+
+  let descr = parquet_reader.metadata().file_metadata().schema_descr_ptr();
+  let num_row_groups = parquet_reader.num_row_groups();
+  let batch_size = 4096;
```

Comment:
Record reader currently uses batch size 256. It is not like it is going to make a difference, just FYI.

Reply:
with 256 it's 20% slower.

Reply:
Then change it so it is comparable!